src/components/CioQuiz/actions.ts

```diff
@@ -34,7 +36,8 @@ export type ActionAnswerQuestion =
   | ActionAnswerInputQuestion
   | Action<QuestionTypes.Cover, CoverQuestionPayload>
   | Action<QuestionTypes.Back>
-  | Action<QuestionTypes.Reset>;
+  | Action<QuestionTypes.Reset>
+  | Action<QuestionTypes.Hydrate, Partial<QuizLocalReducerState>>;
```

Comment:
Yup that's intentional. I had it the way we talked about in the huddle previously (where it resets when a result page is reached) but I thought it would make sense to have it reset only when people click the reset so people could come back to the results.